README.md

```diff
@@ -163,18 +163,18 @@ Offered under the [Apache 2 license][license].
 [blog]: https://buf.build/blog/connect-a-better-grpc
 [buf-studio]: https://buf.build/studio
 [connect-crosstest]: https://github.com/bufbuild/connect-crosstest
-[connect-go]: https://github.com/bufbuild/connect-go
-[connect-protocol]: https://connect.build/docs/protocol
+[connect-go]: https://github.com/connectrpc/connect-go
+[connect-protocol]: https://connectrpc.com/docs/protocol
 [connect-swift]: https://github.com/bufbuild/connect-swift
-[connect-web]: https://www.npmjs.com/package/@bufbuild/connect-web
-[error-handling]: https://connect.build/docs/kotlin/errors
-[getting-started]: https://connect.build/docs/kotlin/getting-started
+[connect-es]: https://www.npmjs.com/package/@connectrpc/connect
+[error-handling]: https://connectrpc.com/docs/kotlin/errors
+[getting-started]: https://connectrpc.com/docs/kotlin/getting-started
 [grpc-protocol]: https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-HTTP2.md
 [grpc-web-protocol]: https://github.com/grpc/grpc-web
-[interceptors]: https://connect.build/docs/kotlin/interceptors
-[license]: https://github.com/bufbuild/connect-go/blob/main/LICENSE
+[interceptors]: https://connectrpc.com/docs/kotlin/interceptors
+[license]: https://github.com/connectrpc/connect-go/blob/main/LICENSE
```

Comment:
I think this should link to the license file in this repo, not the one in connect-go's repo.